lib/counter_culture.rb

```diff
@@ -13,7 +13,13 @@ module ActiveRecord
 
     module ClassMethods
       # this holds all configuration data
-      attr_reader :after_commit_counter_cache
+      def after_commit_counter_cache
+        config = @after_commit_counter_cache
+        if !config && superclass.respond_to?(:after_commit_counter_cache)
+          config = superclass.after_commit_counter_cache
+        end
+        config
+      end
```

Comment:
Actually, now that I think of it, `@@` may not be the solution. I'm not super familiar with how it works, but I think changing it in one place will change it in all the places. So creating a `SimpleReview` will incidentally increment the `twitter_reviews_count`. Not sure, though.